app/models/note.rb

```diff
@@ -4,8 +4,21 @@ class Note < ApplicationRecord
   validates :body, presence: true
   belongs_to :submission
 
-  belongs_to :author,
-             foreign_key: :gravity_user_id,
-             class_name: 'User',
-             primary_key: :gravity_user_id
+  attr_reader :author
+
+  after_initialize :set_author, if: :persisted?
```

Comment:
Not sure if this is the best way to go about things. basically every time we fetch a user (but not after create since that is going to trigger a redirect) we want to fetch their gravity user so we can print a byline,

Reply:
This logic should live in the `user` class and there is already a method that does that: https://github.com/artsy/convection/blob/fd8592f9ba4d2c4a323bb0d7b1489a70691f51ca/app/models/user.rb#L12-L16

So I think it should be a delegation so the amount of extra code would be very minimal and we can consistently use the same logic that does the same thing:

```ruby
delegate :gravity_user, to: :author
```

Also off-topic and not a blocker (but feel free to do so if you can do this quickly), but the result of `User#gravity_user` should probably be memoized to avoid making extra calls to Gravity.

Reply:
The reason for this change is that I learned that admin users do not have a local user record, so we are going straight to gravity for their information.

Reply:
If I had it to do over again, I'd rename the User in this project to Consignor because that's really what it is.